### DISCUSSION TITLE
Unable to open a3p file?

### DISCUSSION THREAD

**Discussion by sbrito** [TheAliceProject/alice3#282] [Q&A] [answered]
One of my students is having trouble opening a file she was working on. 

Any tips on how I might go about helping her open this?
<img width="584" alt="Screen Shot 2022-10-27 at 5 14 04 PM" src="https://user-images.githubusercontent.com/12900936/198399401-68e30e4d-3c61-4d54-8ef3-249e4d1bcd9d.png">

**Comment by dabeshouse** [comment, marked as answer]
There may be earlier versions saved in the directory MidternProject2.bak that can still be opened.
Given the terseness of the message it did not get far in the opening process.
If you want to attach the file I may be able to diagnose or fix it.
